resharper/resharper-unity/src/CSharp/Feature/Services/ContextActions/CreateAssetMenuContextAction.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using JetBrains.Annotations;
+using JetBrains.Application.Progress;
+using JetBrains.Application.UI.Controls.BulbMenu.Anchors;
+using JetBrains.DocumentModel;
+using JetBrains.ProjectModel;
+using JetBrains.ReSharper.Feature.Services.Bulbs;
+using JetBrains.ReSharper.Feature.Services.ContextActions;
+using JetBrains.ReSharper.Feature.Services.CSharp.Analyses.Bulbs;
+using JetBrains.ReSharper.Feature.Services.Intentions;
+using JetBrains.ReSharper.Feature.Services.LiveTemplates.Hotspots;
+using JetBrains.ReSharper.Psi;
+using JetBrains.ReSharper.Psi.CSharp;
+using JetBrains.ReSharper.Psi.CSharp.Tree;
+using JetBrains.ReSharper.Psi.Modules;
+using JetBrains.ReSharper.Psi.Tree;
+using JetBrains.TextControl;
+using JetBrains.Util;
+
+namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.ContextActions
+{
+    [ContextAction(Group = UnityContextActions.GroupID,
+        Name = "Add 'CreateAssetMenu' attribute for scriptable object",
+        Description = "Add 'CreateAssetMenu' attribute for scriptable object which allows to create asset from context menu for that scriptable object")]
```

Comment:
To match existing context actions:

> Add 'CreateAssetMenu' attribute

And:

> Adds the 'CreateAssetMenu' attribute to a scriptable object. This marks a 'ScriptableObject'-derived type to be automatically listed in Unity's 'Assets/Create' menu, so that instances of the type can be easily created and stored in the project as ".asset" files.